explorer/express/vues/index.ejs

```diff
@@ -2,11 +2,11 @@
 <head>
```

Comment:
There is no dynamic content in this file.  
Could we keep it as "index.html", so that it could directly served by nginx ?